src/modules/files.ts

```diff
@@ -113,4 +113,15 @@ export default class Files extends API {
     )
     return response.json()
   }
+
+  /**
+   * Get raw data for a File
+   *
+   * @param file File object
+   * @returns Raw data
+   */
+  async fileData(hash: string): Promise<string> {
+    const response = await this.sendGet(`file/${hash}`)
+    return response.text() as Promise<string>
+  }
```

Comment:
Here is the method that would call this, https://github.com/textileio/go-textile/pull/714

working locally at least!